changelog/58753.fixed

```diff
@@ -0,0 +1 @@
+iRemoving use of undefined varilable in utils/slack.py.
```

Comment:
Drop the initial `i` vim user :)

Reply:
Not a typo.  iRemoving, from Apple, coming this winter.